src/controllers/newConnection/index.ts

```diff
@@ -0,0 +1,48 @@
+import { Get, Produces, Route, SuccessResponse } from 'tsoa'
+import { inject, injectable, singleton } from 'tsyringe'
+
+import { Logger, type ILogger } from '../../logger.js'
+import CompantHouseEntity from '../../models/companyHouseEntity.js'
+import NewConnectionTemplates from '../../views/newConnection.js'
```

Comment:
since you mentioned in the stand up, I follow '-' when it comes to naming files e.g. new-connection.ext and if it would be an instance I would go with the Capital e.g. NewConnection.ext. I guess everybody has got their own preferences here.

I wanted to point out that we already have a view for `Connection` could we put a form in there maybe to keep connection stuff together? Not suggesting to change or anything this would not block me from approving a PR.

Reply:
I'm not sure we have an existing convention on this. @mattdean-digicatapult ?

Reply:
So my reasoning behind creating a newConnections directory for the routes is that if we put them all in the Connections route it would get cluttered, as for each element in HTMX you may need multiple routes/endpoints to perform logic. So my opinion, which probably is flawed, is to put each "VIEW/FORM" in its own organised directory.

As for naming conventions, I looked at some repositories like veritable-cloudagent, and saw that there are some inconsistencies, I'm happy to change to whatever convention the team agrees upon.